### ISSUE TITLE
A very strange bug on apple m1 chip

### ISSUE THREAD

**Issue by vladisa88** [collerek/ormar#119]
I am using ormar with FastAPI. On my Kubuntu 20.04 project runs greatly, but when I try to run the same code on apple m1 chip I am getting an error:

```python
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.9/3.9.2_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/opt/homebrew/Cellar/python@3.9/3.9.2_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/vladislav/.local/share/virtualenvs/test-ormar-SRe1kp3y/lib/python3.9/site-packages/uvicorn/subprocess.py", line 61, in subprocess_started
    target(sockets=sockets)
  File "/Users/vladislav/.local/share/virtualenvs/test-ormar-SRe1kp3y/lib/python3.9/site-packages/uvicorn/server.py", line 49, in run
    loop.run_until_complete(self.serve(sockets=sockets))
  File "/opt/homebrew/Cellar/python@3.9/3.9.2_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/Users/vladislav/.local/share/virtualenvs/test-ormar-SRe1kp3y/lib/python3.9/site-packages/uvicorn/server.py", line 56, in serve
    config.load()
  File "/Users/vladislav/.local/share/virtualenvs/test-ormar-SRe1kp3y/lib/python3.9/site-packages/uvicorn/config.py", line 308, in load
    self.loaded_app = import_from_string(self.app)
  File "/Users/vladislav/.local/share/virtualenvs/test-ormar-SRe1kp3y/lib/python3.9/site-packages/uvicorn/importer.py", line 20, in import_from_string
    module = importlib.import_module(module_str)
  File "/opt/homebrew/Cellar/python@3.9/3.9.2_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "./main.py", line 6, in <module>
    from routers.zoom import zoom_router
  File "./routers/zoom.py", line 8, in <module>
    from models.zoom import Meeting
  File "./models/zoom.py", line 6, in <module>
    from models.license import LicenseAccount
  File "./models/license.py", line 8, in <module>
    class LicenseAccount(ormar.Model):
  File "/Users/vladislav/.local/share/virtualenvs/test-ormar-SRe1kp3y/lib/python3.9/site-packages/ormar/models/metaclass.py", line 502, in __new__
    attrs, model_fields = extract_annotations_and_default_vals(attrs)
  File "/Users/vladislav/.local/share/virtualenvs/test-ormar-SRe1kp3y/lib/python3.9/site-packages/ormar/models/helpers/models.py", line 122, in extract_annotations_and_default_vals
    attrs, model_fields = populate_pydantic_default_values(attrs)
  File "/Users/vladislav/.local/share/virtualenvs/test-ormar-SRe1kp3y/lib/python3.9/site-packages/ormar/models/helpers/pydantic.py", line 127, in populate_pydantic_default_values
    attrs = populate_default_pydantic_field_value(field, field_name, attrs)
  File "/Users/vladislav/.local/share/virtualenvs/test-ormar-SRe1kp3y/lib/python3.9/site-packages/ormar/models/helpers/pydantic.py", line 89, in populate_default_pydantic_field_value
    attrs[field_name] = ormar_field.convert_to_pydantic_field_info()
  File "/Users/vladislav/.local/share/virtualenvs/test-ormar-SRe1kp3y/lib/python3.9/site-packages/ormar/fields/base.py", line 137, in convert_to_pydantic_field_info
    setattr(base, attr_name, cls.__dict__.get(attr_name))
AttributeError: 'FieldInfo' object attribute 'get_constraints' is read-only
```
I actually do not know, if it is related with an apple silicon chip, but I suppose so.

**Comment by GRinvest** [comment]
Fresh install on my Windows system causes the same problem

**Comment by collerek** [comment]
It's connected with pydantic update - fix in 0.9.5.
Please update and check.

**Comment by vladisa88** [comment]
@collerek Now it is working fine, thanks!
